.taskcluster.yml

```diff
@@ -35,20 +35,20 @@ tasks:
         taskId: ${taskGroupId}
         taskGroupId: ${taskGroupId}
         schedulerId: taskcluster-github
-        provisionerId: aws-provisioner-v1
-        workerType: github-worker
+        provisionerId: proj-git-cinnabar
+        workerType: ci
         retries: 5
         scopes:
           $flatten:
             - assume:repo:github.com/${e.base_login}/${e.base_repo_name}:decision-task
             - $if: 'tasks_for == "github-push"'
               then:
-              - 'queue:route:index.github.${e.base_login}.${e.base_repo_name}.*'
-              - secrets:get:repo:github.com/${e.base_login}.${e.base_repo_name}:codecov
+              - 'queue:route:index.project.git-cinnabar.${e.base_login}.*'
+              - secrets:get:project/git-cinnabar/codecov
         routes:
           $if: 'e.decision_id'
           then:
-            - 'index.github.${e.login}.${e.repo_name}.decision.${e.decision_id}'
+            - 'index.project.git-cinnabar.${e.login}.decision.${e.decision_id}'
```

Comment:
but not here.

Reply:
if "not" refers to 'the login part should go away here", then that would cause tags to fail, I think -- the scope and the route would be different.  So, what do you mean?

Reply:
I meant that, but you're right. And `decision_id` is only set under the same condition as the scopes, so it's actually fine to remove the login here too.